src/components/header/index.jsx

```diff
@@ -194,20 +194,21 @@ const NavLink = ({ to, isOpen, route, ...props }) => {
 	}
 
 	function BrandingRedirect(e) {
-		if (to.href == '/' || to.path == '/') {
-			e.preventDefault();
-			location.route('/branding');
-		}
+		e.preventDefault();
+		location.route('/branding');
 	}
+	const homeProps = to.href == '/' || to.path == '/'
+		? { onContextMenu: BrandingRedirect, 'aria-label': 'Home' }
+		: {};
```

Comment:
Better to only attach the handler to the one route it's used, and the `'aria-label': 'Home'` I think is more useful of a description, even if users are probably familiar with the concept that brand/sitename == "home"